.github/workflows/standard_spa_pr.yml

```diff
@@ -0,0 +1,128 @@
+name: Standard SPA PR
+
+on:
+  workflow_call:
+    inputs:
+      use_chromatic:
+        required: true
+        type: boolean
+      use_magic_url:
+        required: true
+        type: boolean
+    secrets:
+      CHROMATIC_PROJECT_TOKEN:
+        required: false
+      AWS_ROLE:
+        required: false
+      AWS_REGION:
+        required: false
+      AWS_APPS_BUCKET:
+        required: false
+
+# Save Money :money_with_wings:
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.repository.name }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  cache-yarn:
+    name: Cache Yarn
+    runs-on: jupiterone-dev
```

Comment:
This is a valid label for a self-hosted runner in `jupiterone-dev`. FYI these runners will be migrated from the existing cluster to a new cluster at some point, but the label will remain the same